frontend/components/branchViewLink.js

```diff
@@ -8,8 +8,8 @@ const isDefaultBranch = (branchName, site) => branchName === site.defaultBranch;
 
 const isDemoBranch = (branchName, site) => branchName === site.demoBranch;
 
-// we only want to link branch names that are alphanumeric plus _ and -
-const isLinkable = s => /^[a-zA-Z0-9_-]+$/.test(s);
+// we only want to link branch names that are alphanumeric plus _, -, and .
+const isLinkable = s => /^[a-zA-Z0-9._-]+$/.test(s);
```

Comment:
I was gonna comment to ask if you needed to escape the dot, but it looks like it works without escaping. Didn't know you could do that.

Reply:
yeah, no escape necessary because it is within a character class (`[...]`). I think our eslint config let me know not to escape.